tests/conf/test_parser.py

```diff
@@ -66,3 +66,17 @@ def test_empty_option(self):
         conf.config_file_path = FN
         conf.read()
         self.assertEqual(conf.reposdir, '')
+
+    def test_iterator_segfault(self):
+        # https://bugzilla.redhat.com/show_bug.cgi?id=2330562
+        empty_conf_path = tests.support.resource_path('etc/empty.conf')
+
+        parser = ConfigParser()
+        parser.read(empty_conf_path)
+        self.assertTrue(parser.hasSection("main"))
+
+        sectObj = parser.getData()["main"]
+        [item for item in sectObj]
```

Comment:
this should not segfault either :)

Reply:
Ok, thanks for info. I was testing just with 3.13.0. Will update this.